functions/Get-DbaClusterNode.ps1

```diff
@@ -0,0 +1,116 @@
+function Get-DbaClusterNode {
+    <#
+        .SYNOPSIS
+            Returns the node(s) of a SQL Cluster.
+
+        .DESCRIPTION
+            Returns the name of the current node(s) in the SQL Server cluster.
+
+            If the -ActiveNode Parameter is passed it only returns the name of the Server currently hosting the clustered instance.
+
+        .PARAMETER SqlInstance
+            Specifies the SQL Server clustered instance to check.
+
+        .PARAMETER SqlCredential
+            Allows you to login to servers using SQL Logins instead of Windows Authentication (AKA Integrated or Trusted). To use:
+
+            $scred = Get-Credential, then pass $scred object to the -SourceSqlCredential parameter.
+
+            Windows Authentication will be used if SqlCredential is not specified. SQL Server does not accept Windows credentials being passed as credentials.
+
+            To connect as a different Windows user, run PowerShell as that user.
+
+        .PARAMETER ActiveNode
+            If this parameter is selected the cmdlet will only return the Active Node in the cluster.
+
+        .PARAMETER Detailed
+            Output all properties, will be deprecated in 1.0.0 release.
+
+        .PARAMETER EnableException
+            By default, when something goes wrong we try to catch it, interpret it and give you a friendly warning message.
+            This avoids overwhelming you with "sea of red" exceptions, but is inconvenient because it basically disables advanced scripting.
+            Using this switch turns this "nice by default" feature off and enables you to catch exceptions with your own try/catch.
+
+        .NOTES
+            Tags:
+            dbatools PowerShell module (https://dbatools.io, clemaire@gmail.com)
+            Copyright (C) 2016 Chrissy LeMaire
+            License: GNU GPL v3 https://opensource.org/licenses/GPL-3.0
+
+        .LINK
+            https://dbatools.io/Get-DbaClusterNode
+
+        .EXAMPLE
+            Get-DbaClusterNode -SqlInstance sqlcluster
+
+            Returns all nodes in the cluster and details about each node.
+
+        .EXAMPLE
+            Get-DbaClusterNode -SqlInstance sqlcluster -ActiveNode
+
+            Returns the name of the active node in the cluster
+
+    #>
+    param (
+        [parameter(Mandatory = $true, ValueFromPipeline = $true)]
+        [Alias("ServerInstance", "SqlServer")]
+        [DbaInstanceParameter]$SqlInstance,
+        [PSCredential]$SqlCredential,
+        [switch]$ActiveNode,
+        [switch]$Detailed,
+        [alias('Silent')]
+        [switch]$EnableException
+    )
+
+    begin {
+        Test-DbaDeprecation -DeprecatedOn 1.0.0 -Parameter Detailed
```

Comment:
Ah, we need to also add a deprecation call for the alias so it spits out the warning as well.

It would just be: `Test-DbaDeprecation -DeprecatedOn "1.0.0" -Alias Get-DbaClusterActiveNode`